docs/src/pages/getting-started/page-layout-examples/sticky-footer/StickyFooter.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/lab/Container';
+
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    minHeight: '100vh',
+  },
+  main: {
+    marginTop: theme.spacing(8),
+    marginBottom: theme.spacing(2),
+  },
+  footer: {
+    padding: theme.spacing(2),
+    marginTop: 'auto',
+    backgroundColor: 'white',
+  },
+});
+
+function StickyFooter(props) {
+  const { classes } = props;
+
+  return (
+    <div className={classes.root}>
+      <CssBaseline />
+      <Container component="main" className={classes.main} maxWidth="sm">
+        <Typography variant="h2" component="h1" gutterBottom>
+          Sticky footer
+        </Typography>
+        <Typography variant="h5" component="h2" gutterBottom>
+          {'Pin a footer to the bottom of the viewport.'}
+          {'The footer will move as the main element of the page grow.'}
+        </Typography>
+        <Typography variant="body1">Fill the sticky footer placeholder.</Typography>
```

Comment:
I'm guessing this is just filler, is there something else we can put here? Or maybe it should be 
```suggestion
        <Typography variant="body1">Sticky footer placeholder.</Typography>
```